go/ephemeral/device_ek.go

```diff
@@ -150,8 +155,8 @@ type DeviceEKsResponse struct {
 	} `json:"results"`
 }
 
-func GetOwnActiveDeviceEKMetadata(ctx context.Context, g *libkb.GlobalContext) (activeDeviceEKMetadata map[keybase1.DeviceID]keybase1.DeviceEkMetadata, err error) {
-	defer g.CTrace(ctx, "GetOwnActiveDeviceEKMetadata", func() error { return err })()
+func GetActiveDeviceEKMetadata(ctx context.Context, g *libkb.GlobalContext) (metadata map[keybase1.DeviceID]keybase1.DeviceEkMetadata, err error) {
```

Comment:
Do you think "own" is redundant, since we only ever do this for self? I could agree with that.

Reply:
yeah it felt a little too verbose